public/pages/Emails/EmailGroups.tsx

```diff
@@ -29,7 +33,7 @@ export function EmailGroups(props: EmailGroupsProps) {
       </EuiTitle>
 
       <EuiSpacer />
-      <RecipientGroupsTable coreContext={coreContext} />
+      <RecipientGroupsTable coreContext={coreContext} notificationService={props.notificationService} />
```

Comment:
What's the benefit that we change move notificationService out from the ServiceContext to a props?